features/git-ship/supplied_branch/no_conflicts_with_conflicting_changes.feature

```diff
@@ -1,74 +0,0 @@
-Feature: git ship: shipping the supplied feature branch (with conflicting changes)
-
-  As a developer getting the LGTM for a feature branch while working on unrelated things that conflict with the main branch
```

Comment:
I deleted two specs where we had with_conflicting_changes, with_open_changes, and without_open_changes. I moved to with_open_changes (which are now conflicting) and without_open_changes. I decided I agree with you on #176.

Otherwise all the other specs are simply merged into other files. These specs are now at
```
features/git-ship/supplied_branch/feature_branch/with_tracking_branch.feature
features/git-ship/supplied_branch/feature_branch/without_tracking_branch.feature
```

If you are worried about any other specific specs please point them out and I will let you know where they now live.